sdks/python/apache_beam/io/gcp/bigquery_test.py

```diff
@@ -566,15 +563,13 @@ def test_no_table_and_create_never(self, patched_time_sleep):
     client.tables.Get.side_effect = HttpError(
         response={'status': '404'}, url='', content='')
     create_disposition = beam.io.BigQueryDisposition.CREATE_NEVER
-    with self.assertRaises(RuntimeError) as exn:
+    with self.assertRaisesRegexp(
+        RuntimeError, r'Table project:dataset.table not found but create '
+                      r'disposition is CREATE_NEVER.'):
```

Comment:
<!--new_thread; commit:4709161da347f4fa5eedbd7f3c09911b573fadac; resolved:0-->
Let's add `^` and `$` and escape the '.' characters throughout.

Reply:
done